.ci/install_proxy.sh

```diff
@@ -23,10 +23,24 @@ source "${cidir}/lib.sh"
 
 clone_build_and_install "github.com/clearcontainers/proxy"
 start_proxy_cmd="sudo systemctl start cc-proxy"
+upstart_source_file="${cidir}/data/cc-proxy.conf"
+upstart_destr_file="/etc/init/cc-proxy.conf"
+
+
+# If we are running under the metrics CI system then we do not want the proxy
+# to be dynmaically changing the KSM settings under us - we need control of them
+# ourselves
+if [[ $METRICS_CI ]]; then
+	proxy_systemd_file=$(sudo systemctl show cc-proxy | fgrep FragmentPath | awk 'BEGIN{FS="="}{print $2}')
+	sudo sed -i 's/\(^ExecStart.*$\)/\1 -ksm initial/' "${proxy_systemd_file}"
```

Comment:
what about also adding `-log debug` here?  Just as in `cc-proxy.conf.noksm` file

Reply:
I did notice that we had `-log` in the upstart files we inject, but did not inject it into the systemd files installed by the package already - and wondered if that disparity was deliberate or not?
As the current code did not add `-log` to the systemd files, I went with the same methodology.
@chavafg - what do you think, maybe we should always add `-log` for both upstart and systemd - in which case I'm happy to make the change

Reply:
Hi @grahamwhaley , I think we didn't added the `-log` option simply because we forgot it and yes, we should always add the option. Please make the change :) 
Thanks.

Reply:
@chavafg added and pushed - please review, as not run-tested here and I suspect CI may only run-test half of the code (not sure if we cover both upstart and systemd).
Note, sed expr param changed from 'x' to "x" as we now use an expansion to build the args.